approve-n-merge-queue/src/main.ts

```diff
@@ -206,20 +188,7 @@ async function main() {
     core.debug('skip approving this PR')
   }
 
-  const mergeQueueState = pr.mergeQueueEntry?.state ?? 'MERGEABLE'
-
-  if (mergeQueueState === 'MERGEABLE' || mergeQueueState === 'AWAITING_CHECKS') {
-    core.debug('time to enqueue this PR')
-    const enqueueResult = await enqueuePR(kit, { prId: pr.id })
-
-    if (!enqueueResult.enqueued) {
-      core.setFailed(`Could not enqueue PR #${pr.number}`)
-      return
-    }
-  } else {
-    console.log(pr)
-    core.setFailed('PR seems to be unmergeable')
-  }
+  await autoMergePr(kit, { prId: pr.id })
```

Comment:
so instead we just enable auto merge here, this way we don't need to "catch" the right moment when to manually enqueue the PR